nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/test/java/org/nuxeo/ecm/platform/comment/TestCommentsMigrator.java

```diff
@@ -241,33 +206,29 @@ public void testProbe() {
 
         CommentServiceConfig config = newConfig();
         CommentManager relationCommentManager = new CommentManagerImpl(config);
-        CommentManager propertyCommentManager = new PropertyCommentManager();
 
         DocumentModel domain = session.createDocumentModel("/", "test-domain", "Domain");
         session.createDocument(domain);
-        DocumentModel file = session.createDocumentModel("/test-domain", "file1", "File");
+        DocumentModel file = session.createDocumentModel("/test-domain", "anotherFile", "File");
```

Comment:
This became complex to understand because you're ending with tests leveraging `@Before` setup and some tests don't.

I would rather keep the design of the test which is shared between all migrator tests.

Reply:
Before our new devs, the two methods (`testMigration` and `testMigrationThroughService`) create a relation comments in the runMigration method, the `testProbe` in other hand doesn't rely on the run migration but create his own data.  After refactoring, it leverages the ones in the before like the 2 others